app/controllers/search_controller.rb

```diff
@@ -87,9 +87,13 @@ def users
   end
 
   def usernames
-    usernames = Search::User.search_usernames(params[:username])
+    result = if FeatureFlag.enabled?(:search_2_usernames)
+               Search::Postgres::Username.search_documents(params[:username])
```

Comment:
I'm not sweating the naming here just yet. I suspect there may be an opportunity to refactor this into a singular `Search::Postgres::User` namespace once we implement `/search/users`.